.buildkite/pipeline.yml

```diff
@@ -15,9 +18,12 @@ steps:
     plugins:
       - JuliaCI/julia#v1:
           version: "1.6-nightly"
-      - JuliaCI/julia-test#v1:
       - JuliaCI/julia-coverage#v1:
           codecov: true
+    commands:
+      - julia --project=test -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd()))'
+      - julia --project=test -e 'using Pkg; Pkg.develop(PackageSpec(path=joinpath(pwd(),"lib","CUDAKernels")))'
+      - julia --project=test --color=yes --check-bounds=yes --code-coverage=user --depwarn=yes test/runtests.jl
```

Comment:
Just for the record: it should be possible to put back `JULIA_LOAD_PATH=@` once a fix like https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1007 is implemented

(But using `JULIA_LOAD_PATH=@` was just my preference for making sure that test is reproducible; i.e., avoid accidentally using `@stdlib` and `@v#.#`. It's not a strict requirement to make CI work.)